src/receivers/ExpressReceiver.ts

```diff
@@ -4,6 +4,7 @@ import { createServer, Server, ServerOptions } from 'http';
 import { createServer as createHttpsServer, Server as HTTPSServer, ServerOptions as HTTPSServerOptions } from 'https';
 import { ListenOptions } from 'net';
 import express, { Request, Response, Application, RequestHandler, Router } from 'express';
+import * as core from 'express-serve-static-core';
```

Comment:
Did this dep need to get npm installed? I don't see any changes to `package.json`.